shub/tool.py

```diff
@@ -52,3 +53,6 @@ def cli():
     command_module = importlib.import_module(module_path)
     command_name = command.replace('_', '-')  # easier to type
     cli.add_command(command_module.cli, command_name)
+
+# Add image commands group
+cli.add_command(image_cli, "image")
```

Comment:
Wouldn't this work without extra code (i.e. by just adding `image` to the `commands` list) if we rename `image_cli` to `cli` in `shub/image/__init__.py`?)